parts/common/components.json

```diff
@@ -1117,23 +1117,56 @@
       }
     },
     {
-      "name": "cni-plugins",
+      "name": "containernetworking-plugins",
       "downloadLocation": "/opt/cni/downloads",
       "downloadURIs": {
         "windows": {
           "default": {
             "versionsV2": []
           }
         },
+        "ubuntu": {
+          "r1804": {
```

Comment:
this seems highly repetitive can't i have a default ubuntu version and only be differnet if I actually need a different version?

Same questions as @Sylvain had what about mariner?